app/helpers/image_encode_helper.rb

```diff
@@ -0,0 +1,33 @@
+module ImageEncodeHelper
+  def encodable_image_tag(source, options = {})
+    image_tag(encodable_image_source(source), options)
+  end
+  alias eimage_tag encodable_image_tag
+
+  def encodable_image_source(source)
+    if base64_encode_images? && source.present?
+      base64_encoded_uri(source)
+    else
+      path_to_image(source)
+    end
+  end
+  alias eimage_source encodable_image_source
+
+  def base64_encode_images?
+    @base64_encode_images
+  end
+
+  def base64_encoded_uri(source)
+    asset = Rails.application.assets[source]
+
+    if asset.content_type == 'image/svg+xml'
```

Comment:
Yeah, Prince prefers the straight SVG sometimes. 

**Not encoded:**
<img width="115" alt="screen shot 2017-06-01 at 4 59 44 pm" src="https://cloud.githubusercontent.com/assets/39493/26700660/2bf30b46-46ec-11e7-9c60-5e49e87a2cfa.png">

**Encoded**
<img width="113" alt="screen shot 2017-06-01 at 5 03 48 pm" src="https://cloud.githubusercontent.com/assets/39493/26700696/4e11dc48-46ec-11e7-8f9e-9280fdc5f950.png">

Plus, it's [not _always_ better](https://css-tricks.com/probably-dont-base64-svg/).